webui/react/src/components/kit/Pivot.tsx

```diff
@@ -1,20 +1,21 @@
 import { Tabs } from 'antd';
-import React, { ReactNode } from 'react';
+import React, { CSSProperties, KeyboardEvent, MouseEvent, ReactNode } from 'react';
 
 export type TabItem = {
   children?: ReactNode;
   key: string;
-  label: string;
+  label: ReactNode;
 };
 
 interface PivotProps {
   activeKey?: string;
   defaultActiveKey?: string;
   destroyInactiveTabPane?: boolean;
   items?: TabItem[];
-  onChange?: () => void;
-  onTabClick?: () => void;
+  onChange?: (activeKey: string) => void;
+  onTabClick?: (key: string, event: MouseEvent | KeyboardEvent) => void;
   tabBarExtraContent?: ReactNode;
+  tabBarStyle?: CSSProperties;
```

Comment:
This component should not take arbitrary CSS. If there are additional variants needed then the UI kit should be updated to address that. (likewise, if we need to update padding that should be baked into the component.

Reply:
Does add padding to the left considered as an additional variant, or we should always put padding on the outside component? 
<img width="1082" alt="Screen Shot 2023-01-10 at 10 07 12 AM" src="https://user-images.githubusercontent.com/40620519/211602164-66d933cf-b4fe-4e59-9e22-019fa3f8ed84.png">

Reply:
It's hard to say exactly based on one example but I think in this most likely the layout component here should have padding.

Reply:
So on this page we're manually overriding the page's padding because we're dropping the special header in there. The issue we're running into is that `<ModelVersionHeader` is really part of the _layout_ but it gets dropped into the content portion of `<Page>` because we don't have standardized layout handling.

I think for this PR, the correct approach is to add an extra `<div>` that has the necessary padding and leave a `TODO` to clean that up once we standardize page layouts.

Reply:
Got it, thanks!